### DISCUSSION TITLE
Any status on current work in the Design tab?

### DISCUSSION THREAD

**Discussion by OneNiko727** [ppy/osu#29091] [Q&A] [answered]
osu! lazer is very much able to replace stable now but even after 6 years we still do not have a storyboard editor.
I haven't been able to find much information on the design tab but at least we can edit storyboards with storybrew now that we have the ability to edit maps externally.

Any info would be amazing to hear

**Comment by peppy** [comment, marked as answer]
Nothing yet sorry. This editor in itself will be a very large project and we're focused on making everything else first.

As already suggested, you can either use stable or storybrew in the mean time (which is why this isn't getting super high priority in the first place).

**Reply by OneNiko727** [reply]
Alright I understand
